lua/gatekeeper.lua

```diff
@@ -99,6 +99,11 @@ local ffi = require("ffi")
 -- Structs
 ffi.cdef[[
 
+struct constants {
+	static const uint32_t BONDING_MODE_ROUND_ROBIN = 0;
+	static const uint32_t BONDING_MODE_8023AD = 4;
```

Comment:
I think it's a good idea to add the other modes since they only require the constants in Lua.